src/modules/pair/services/pair.compute.service.ts

```diff
@@ -543,4 +555,102 @@ export class PairComputeService implements IPairComputeService {
                 .multipliedBy(firstTokenPriceUSD);
         }
     }
+
+    @ErrorLoggerAsync({
+        logArgs: true,
+    })
+    @GetOrSetCache({
+        baseKey: 'pair',
+        remoteTtl: CacheTtlInfo.ContractState.remoteTtl,
+        localTtl: CacheTtlInfo.ContractState.localTtl,
+    })
+    async hasFarms(pairAddress: string): Promise<boolean> {
+        return await this.computeHasFarms(pairAddress);
+    }
+
+    async computeHasFarms(pairAddress: string): Promise<boolean> {
+        const addresses: string[] = farmsAddresses(['v2']);
```

Comment:
use `FarmVersion` enum instead of string

Reply:
done